### ISSUE TITLE
shfmt fails when running with `--env production`

### ISSUE THREAD

**Issue by djerihovs** [DannyBen/bashly#443]
Hello,

After generation of script with production env flag and config added shfmt fails because bashly generate adds unnecessary new lines before functions config_show() and  ini_load()

How to reproduce:

```
bashly init --minimal
bashly add config
bashly generate --env production
shfmt --diff --case-indent --indent 2  download 
```

shfmt Output:
```
--- download.orig
+++ download
@@ -118,7 +118,6 @@
 
 }
 
-
 config_show() {
   config_load
   ini_show
@@ -177,7 +176,6 @@
   ini_save "$CONFIG_FILE"
 }
 
-
 ini_load() {
   declare -gA ini
 ```

**Comment by DannyBen** [comment]
Thanks.

This happens because of the empty lines in between comment blocks in the two files that are added by `bashly add config`:

https://github.com/DannyBen/bashly/blob/1d4306e5a2754b5f2d972806f5e34ddfdfb28285/lib/bashly/libraries/config/config.sh#L10-L12

https://github.com/DannyBen/bashly/blob/1d4306e5a2754b5f2d972806f5e34ddfdfb28285/lib/bashly/libraries/ini/ini.sh#L27-L29

You can remove them in your copy. I will update the files for the next release.

**Comment by DannyBen** [comment]
This is now fixed in version 1.1.2.
Thanks for reporting.
